bin/util.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import print_function
```

Comment:
Ah - Anaconda does create an alias 'python3' for that Python.  Sure, we 
can do that.